src/elements/content-explorer/ContentExplorer.js

```diff
@@ -435,6 +476,7 @@ class ContentExplorer extends Component<Props, State> {
         const hasFolderChanged = currentId && currentId !== folderId;
         const hasSearchQuery = !!searchQuery.trim().length;
         const offset = hasFolderChanged || hasSearchQuery ? 0 : currentOffset; // Reset offset on folder or mode change
+        const shouldFetchThumbnails = getProp(features, 'contentExplorer.gridView.enabled');
```

Comment:
I think there is a `isFeatureEnabled` helper available in `feature-checking/util`.